plutus-use-cases/src/Plutus/Contracts/Crowdfunding.hs

```diff
@@ -150,15 +152,15 @@ scriptInstance = Scripts.validatorParam @Crowdfunding
 validRefund :: Campaign -> PubKeyHash -> TxInfo -> Bool
 validRefund campaign contributor txinfo =
     -- Check that the transaction falls in the refund range of the campaign
-    Interval.contains (refundRange campaign) (txInfoValidRange txinfo)
+    Interval.contains (refundRange campaign) (Ledger.posixTimeRangeToSlotRange $ txInfoValidRange txinfo)
```

Comment:
ditto

Reply:
I'm going to stop saying this: basically all the configuration in the usecases should be using times.